mkdocs.yml

```diff
@@ -8,15 +8,17 @@ markdown_extensions:
   - attr_list
   - admonition
   - def_list
-  - codehilite(guess_lang=false)
-  - toc(permalink=true)
+  - codehilite:
+      guess_lang: false
+  - toc:
+      permalink: true
   - pymdownx.superfences
   - pymdownx.inlinehilite
   - pymdownx.tilde
   - pymdownx.betterem
   - pymdownx.mark
 
-pages:
+nav:
```

Comment:
@joannechester - What's the symptom if this is left as `pages`? Does it fail, or does it just a warning?

(My understanding from the mydocs release notes is that it's *supposed* to show a warning.)

Reply:
It is just a warning at this stage.  I have reverted that change.